### DISCUSSION TITLE
Range thinner and sleeker config || add Moz Thumb to Progress

### DISCUSSION THREAD

**Discussion by half-metal** [saadeghi/daisyui#2122] [Ideas / feature requests]
This would be an enhancement to the DaisyUI range or progress bar - allowing the range to be thinner where a configurable moz range thumb does not look weird, also maybe configurable moz range thumb color. Below is a screenshot, at the top is a custom range, and below are the daisyui range.

<img width="309" alt="Screen Shot 2023-01-03 at 12 55 22 PM" src="https://user-images.githubusercontent.com/58837299/210440291-f7bc8004-4d28-4ed2-88c1-0fa37a9b1be1.png">

Yes, this can be custom made, but I imagine this would be reused by many DaisyUI folks as this is very common in music players. Right now, the range seems fat and large to what is actually needed for tracking location in a track. Where something more minimal could be used. When you adjust the height on the DaisyUI range also, it starts to look weird like Pong.

<img width="305" alt="Screen Shot 2023-01-03 at 1 05 06 PM" src="https://user-images.githubusercontent.com/58837299/210441344-48f8c8e4-85f9-46b9-b80e-7d861a471dba.png">

An alternative could be just adding a moz range thumb to the current DaisyUI Progress, and could emulate this common look in music players.

**Comment by saadeghi** [comment]
We should have the same look with both webkit and moz. There are different pseudo selectors for each one and they work differently.  
Also, I want it to have different colors (for the part that is empty and the part that is filled)

If you know a way to achieve this with CSS only let me know because as far as I know, there is no cross-browser, CSS-only way (yet) to have a narrow bar with 2 colors and a big thumb.

PS: I'm moving this to discussions
